liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -217,16 +220,18 @@ protected void setContainer(boolean container) {
     private class DevMojoUtil extends DevUtil {
 
         Set<String> existingFeatures;
+        Map<String, File> libertyDirPropertyFiles = new HashMap<String, File> ();
 
-        public DevMojoUtil(File serverDirectory, File sourceDirectory, File testSourceDirectory, File configDirectory, File projectDirectory,
+        public DevMojoUtil(File installDir, File userDir, File serverDirectory, File sourceDirectory, File testSourceDirectory, File configDirectory, File projectDirectory,
                 List<File> resourceDirs) throws IOException {
             super(serverDirectory, sourceDirectory, testSourceDirectory, configDirectory, projectDirectory, resourceDirs, hotTests,
                     skipTests, skipUTs, skipITs, project.getArtifactId(), serverStartTimeout, verifyTimeout, verifyTimeout,
                     ((long) (compileWait * 1000L)), libertyDebug, false, false, pollingTest, container, dockerfile, dockerRunOpts, 
                     dockerBuildTimeout, skipDefaultPorts);
 
             ServerFeature servUtil = getServerFeatureUtil();
-            this.existingFeatures = servUtil.getServerFeatures(serverDirectory);
+            this.libertyDirPropertyFiles = BasicSupport.getLibertyDirectoryPropertyFiles(installDirectory, userDirectory, serverDirectory);
+            this.existingFeatures = servUtil.getServerFeatures(serverDirectory, libertyDirPropertyFiles);
```

Comment:
I just noticed that this constructor's parameters are `installDir` and `userDir`, but the constructor body uses `installDirectory` and `userDirectory`.

Reply:
good catch...it must be getting those from the StartDebugMojoSupport class hierarchy on DevMojo. I'll change to use the ones passed in.